src/Strava/API/Service/REST.php

```diff
@@ -92,14 +105,24 @@ public function getAthlete($id = null)
             $path = 'athletes/' . $id;
         }
         $parameters['query'] = ['access_token' => $this->getToken()];
-        return $this->getResponse('GET', $path, $parameters);
+
+        $response = $this->getResponse('GET', $path, $parameters);
+
+        if ($this->responseVerbosity == 0)
```

Comment:
I think these comparison operators could be type-safe "===" instead of "==".

Reply:
I agree, now fixed in a new commit